pkg/cloud/azure/azurestorageintegration.go

```diff
@@ -0,0 +1,99 @@
+package azure
+
+import (
+	"strings"
+	"time"
+
+	"github.com/opencost/opencost/pkg/cloud"
+	"github.com/opencost/opencost/pkg/kubecost"
+	"github.com/opencost/opencost/pkg/util/timeutil"
+)
+
+type AzureStorageIntegration struct {
+	AzureStorageBillingParser
+	ConnectionStatus cloud.ConnectionStatus
+}
+
+func (asi *AzureStorageIntegration) GetCloudCost(start, end time.Time) (*kubecost.CloudCostSetRange, error) {
+	computeCCSR, err := kubecost.NewCloudCostSetRange(start, end, timeutil.Day, asi.Key())
```

Comment:
Can we rename this variable to just `ccsr`

Reply:
Rename done